src/client/components/Footer/footer.css

```diff
@@ -0,0 +1,48 @@
+footer {
+  width: 100%;
+  background-color: #929B90;
+}
+
+footer .footer {
+  display: flex;
+  justify-content: right;
+  align-items: center;
+  padding-right: 10%;
+}
+
+footer .footer .column {
+  padding: 4%;
+}
+
+footer h3 {
+  color: white;
+}
+
+footer ul {
+  list-style-type: none;
+  display: inline;
+}
+
+footer li {
+  line-height: 28px;
```

Comment:
When it comes to font related properties try to use em as units, rather than pixels. You can read more about it here > [mdn](https://developer.mozilla.org/en-US/docs/Web/CSS/font-size)

Reply:
You are right. I updated that. Thanks :)